c2corg_ui/static/js/map/map.js

```diff
@@ -413,6 +413,8 @@ app.MapController.prototype.createStyleFunction_ = function() {
           case 'profiles':
             return this.createPointStyle_(feature, resolution);
           case 'routes':
+          case 'xreports':
+            return this.createPointStyle_(feature, resolution);
```

Comment:
This is not correct => you have changed the switch for routes and duplicated the ``return this.createPointStyle_(feature, resolution);`` whereas it already exists 2 lines above.

Please cancel this change and simply add ``case 'xreports':`` after line 413.